pISSStream/View/PissLabel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  PissLabel.swift
+//  pISSStream
+//
+//  Created by durul dalkanat on 12/25/24.
+//
+
+import SwiftUI
+
+struct PissLabel: View {
+    let amount: String
+    let isConnected: Bool
+    
+    var body: some View {
+        HStack(spacing: 4) {
+            Circle()
+                .fill(isConnected ? Color.green : Color.red)
+                .frame(width: 8, height: 8)
```

Comment:
This doesn't show up on my 14.7.1 system:

![Screenshot](https://github.com/user-attachments/assets/6cc4873b-9837-4580-8252-c5b7095a5cff)

Should this work on eg. macOS 15.x?

Reply:
Eh you know what, I'm just going to go ahead and merge this, maybe with a change so that the emojis change to eg. 🧑🏽‍🚀❗ on LOS. We can worry about details later 😀

Reply:
> This doesn't show up on my 14.7.1 system:
> 
> ![Screenshot](https://private-user-images.githubusercontent.com/191785648/398599157-6cc4873b-9837-4580-8252-c5b7095a5cff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUxNjc5ODEsIm5iZiI6MTczNTE2NzY4MSwicGF0aCI6Ii8xOTE3ODU2NDgvMzk4NTk5MTU3LTZjYzQ4NzNiLTk4MzctNDU4MC04MjUyLWM1YjcwOTVhNWNmZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNVQyMzAxMjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMzEyN2ViMjZhODFjM2JmMTVkYjNhNjIxNDUzNjc0NzEyZmIwMjYxMDZmMDkyNDZlZTlkODJlNzY0YTAyMjBjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.AUIyKKwswzdUHMlgfO7z3n_50Ie9dyrwiWfNwgYezs8)
> 
> Should this work on eg. macOS 15.x?

Yes I am using 15.2 (24C101). Here is my screenshot

![Screenshot 2024-12-25 at 5 04 39 PM](https://github.com/user-attachments/assets/0fad0ac4-efa3-4ef6-a4c6-acbbf97e7ba0)

Reply:
another one 

![Screenshot 2024-12-25 at 5 08 02 PM](https://github.com/user-attachments/assets/fa3f7d73-88e7-4b9d-b944-691e9a89e64d)